src/components/browse/singleDatasetView/singleDatasetView.tsx

```diff
@@ -0,0 +1,223 @@
+import {
+  Accordion,
+  Col,
+  Container,
+  Row,
+  Tab,
+  Table,
+  Tabs,
+} from "react-bootstrap";
+
+const SingleDatasetView = () => {
+  return (
+    <Container className="py-4">
+      <h5>
+        <strong>
+          ZZZ Coverage bias sensitivity of variant calling_ for 4 WG_seg_tech
+        </strong>
+      </h5>
+      <p>Dataset ID | ZZZ EGAD00001000174</p>
+      <p className="fs-7">
+        <span>Study Type | ZZZ Badges </span>
+        <span>Centre name: ZZZ Centre</span>
+      </p>
+      <Row className="fs-7">
+        <Col>Description</Col>
+        <Col className="text-end">Status Badge</Col>
+      </Row>
+      <Row className="fs-8">
+        <Col>
+          ZZZ SET Coverage bias sensitivity of variant calling for 4 WG seq tech
+          DATA SET Coverage bias sensitivity of variant calling for 4 WG seq
+          tech DATA SET Coverage bias sensitivity of variant calling for 4 WG
+          seq tech DATA SET Coverage
+        </Col>
+      </Row>
+      <Row className="fs-8 mb-4">
+        <Col className="text-end">Date</Col>
+      </Row>
+      <Container className="mb-5">
+        <Tabs className="mb-4">
```

Comment:
In the console, there are 2 warnings in total. 
1. `<div> cannot appear as a descendant of <p>`
2. `<tr> cannot appear as a child of <table>. Add a <tbody>, <thead> or <tfoot> to your code to match the DOM tree generated by the browser.`